examples/Tutorials/Basic Button/Podfile

```diff
@@ -0,0 +1,21 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'JustRide' do
+  # Comment this line if you're not using Swift and don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for JustRide
+  pod 'UberRides'
```

Comment:
Probably want to use `pod 'UberRides', '~> 0.5'` to ensure that these will work even after a major version bump